app/src/test/kotlin/co/netguru/baby/monitor/client/feature/client/home/RestartAppUseCaseTest.kt

```diff
@@ -0,0 +1,43 @@
+package co.netguru.baby.monitor.client.feature.client.home
+
+import android.content.Context
+import android.content.Intent
+import android.content.pm.PackageManager
+import androidx.appcompat.app.AppCompatActivity
+import co.netguru.baby.monitor.RxSchedulersOverrideRule
+import com.nhaarman.mockito_kotlin.doReturn
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.verify
+import org.junit.Rule
+import org.junit.Test
+
+class RestartAppUseCaseTest {
+
+    @Rule
+    @JvmField
+    val schedulersRule = RxSchedulersOverrideRule()
+
+    private val restartAppUseCase: RestartAppUseCase = RestartAppUseCase()
+    private val intent: Intent = mock()
+    private val packageName = "packageName"
+    private val activity: AppCompatActivity = mock {
```

Comment:
Suggestion: If testing things connected to android framework will get too complicated consider using robolectric http://robolectric.org/